go/oasis-test-runner/oasis/client.go

```diff
@@ -35,6 +35,13 @@ type ClientCfg struct {
 	RuntimeConfig      map[int]map[string]interface{}
 }
 
+// ConfigureNewRuntime configures client node with a new runtime.
+//
+// Warning: In order to apply configuration changes node has to be restarted.
+func (client *Client) ConfigureNewRuntime(rtIdx int) {
```

Comment:
Warning looks too scary. This should be a note, but I would prefer to remove this part.
The comment is not totaly clear, since you are accepting index and not runtime. Will this also work if we configure the same index twice?
Why don't we keep the same name as in compute worker?